packages/language-chooser/src/language-chooser/active-controls.tsx

```diff
@@ -0,0 +1,92 @@
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+import { Button, ButtonGroup } from '@wordpress/components';
+
+interface ActiveControlsProps {
+	onMoveUp: () => void;
+	onMoveDown: () => void;
+	onRemove: () => void;
+	isMoveUpDisabled: boolean;
+	isMoveDownDisabled: boolean;
+	isRemoveDisabled: boolean;
+}
+function ActiveControls( {
+	onMoveUp,
+	onMoveDown,
+	onRemove,
+	isMoveUpDisabled,
+	isMoveDownDisabled,
+	isRemoveDisabled,
+}: ActiveControlsProps ) {
+	return (
+		<ButtonGroup>
```

Comment:
We're actually in the process of deprecating `ButtonGroup` ([context](https://github.com/WordPress/gutenberg/issues/65339)), so it would be better to replace it with another component (maybe a `VStack` or a simple vanilla HTML element).

Reply:
Of course it is. I just switched to it after it was recommended to use it. Guess I‘ll switch to something else now.

Reply:
@swissspidy feel free to blame me for this one. 

I recommended using `ButtonGroup` as a better alternative to an unordered list, but I did it before we decided that `ButtonGroup` would be deprecated.